README.md

```diff
@@ -130,3 +130,62 @@ To debug native code with VSCode or CLion or any other IDE:
 3. Set the parameters to be the ones used by the ```mvn``` script, as per above
 4. Set the working directory to the `aws-crt-java` directory
 5. On windows, you will need to manually load the PDB via the Modules window in Visual Studio, as it is not embedded in the JAR. It will be in the ```target/cmake-build/lib/windows/<arch>``` folder.
+
+## Platform-Specific JARs
+
+Since v0.20.5, platform-specific JARs are available to reduce the size of the artificts. To consume the platform-specific JAR of aws-crt-java, you can use Maven classifier (https://maven.apache.org/pom.html) property to specify the platform. Without classifier, you will get the uber jar to support all distributions.
+
+``` xml
+        <!-- Linux x86_64 distribution  -->
```

Comment:
```suggestion
        <!-- Platform-specific Linux x86_64 JAR -->
```

I didn't like introducing a new word ~distribution~ when we're already using the term "platform"

Just trying to reduce the number of different words we use to describe "platform-specific" (distribution, platform, OS, architecture, classifier). Since Java devs don't usually need to think about that kind of stuff